integration-tests/typescript-koa/src/api.github.com.yaml/models.ts

```diff
@@ -5878,7 +5878,7 @@ export type t_validation_error = {
     index?: number
     message?: string
     resource?: string
-    value?: string | null | number | string[]
+    value?: (string | null) | (number | null) | string[]
```

Comment:
Technically these `()` are superfluous, but probably not worth trying to change right now